components/EventForm/index.js

```diff
@@ -0,0 +1,101 @@
+import { StyledForm, StyledHeading, StyledLabel, StyledInput, StyledSaveButton, StyledCldUploadButton, StyledGoBackButton } from "./EventForm.styled";
+import StyledDatePicker from "./StyledDatePicker";
+import { useState } from "react";
+import Image from "next/image";
+import useSWR from "swr";
+
+
+export default function EventForm({ kidData }) {
```

Comment:
Currently, you don't need to pass the `kidData` to the form. You're not using it except for extracting the `id` and the events from it.

Reply:
Thanks for your Input. I changed most of it, but now adding an event doesn´t work anymore. Could you have another look at it and tell me what I did wrong, please?

Reply:
I fixed it now. :-)